web/src/components/insight-page/index.tsx

```diff
@@ -69,10 +88,8 @@ export const InsightIndexPage: FC = memo(function InsightIndexPage() {
         gridTemplateColumns="repeat(2, 1fr)"
         mt={2}
       >
-        <DeploymentFrequencyChart data={data} />
-        <ChangeFailureRateChart data={[]} />
-        <LeadTimeForChangesChart data={[]} />
-        <MeanTimeToRestoreChart data={[]} />
```

Comment:
Do we plan to remove those two charts, or just disable them for now because of no valid data provided currently? I saw this PR deleted those components.

Reply:
Yes, they will be re-added in the future. So deleted them now to be simple.